.pipelines/onebranch/templates/template-buildrp-buildaro.yml

```diff
@@ -9,6 +9,12 @@ steps:
       export COMMIT=$(git rev-parse --short=7 HEAD)$([[ $(git status --porcelain) = "" ]] || echo -dirty)
       if [ -z "$TAG" ];
       then
+        if [ -z "$is_official_release"]
+        then
+          git describe --exact-match
+          echo "Ensure there is an annotated tag (git tag -a) for git commit $(COMMIT)"
+          exit 1
```

Comment:
I'm not sure if I'm missing something, but doesn't this make if `is_official_release == true` then it will always `exit 1`?

Reply:
It's a bit of action-at-a-distance: you can only get here (past the test on line 10) if $TAG didn't get set to anything up on line 2, because git failed to resolve an (annotated) tag for the current revision.